dotfiles/Brewfile

```diff
@@ -1,29 +1,25 @@
 cask_args appdir: '/Applications'
 
 tap 'homebrew/services'
-tap 'caskroom/cask'
+tap 'homebrew/cask'
 
-brew 'wget'
+brew 'ack'
+brew 'asdf'
 brew 'coreutils'
-brew 'moreutils'
 brew 'findutils'
 brew 'git'
-brew 'vim'
-brew 'ack'
-brew 'the_silver_searcher'
-brew 'watchman'
-brew 'node'
-brew 'yarn'
-brew 'nvm'
 brew 'hub'
-brew 'elixir'
-brew 'postgresql'
-brew 'redis'
-brew 'ctags'
 brew 'libyaml'
+brew 'the_silver_searcher'
+brew 'tmux'
+brew 'tree'
+brew 'vim'
+brew 'volta'
+brew 'watchman'
+brew 'wget'
 brew 'zsh'
 
 cask 'google-chrome'
+cask 'iterm2'
```

Comment:
Maybe add firefox for good measure?

Reply:
I prefer firefox-developer-edition which is a different cask. So it seems to me to leave that up to the user to decide for themselves.

Reply:
Ditto on Developer Edition. Chrome's the "standard" so my vote's for letting folks add their preference(s) after the fact.